.gitignore

```diff
@@ -1,5 +1,6 @@
 # Generic Files
 *.txt
 node_modules
-**/contentBundle.js
-**/contentBundle.js.map
\ No newline at end of file
+dist
+!dist/manifest.json
```

Comment:
do we need to ignore this if we already ignore dist

Reply:
That is un-ignoring the manifest.json in the dist folder. If we end up having multiple bundle / output files from webpack (One for popup, one for background and more), then we won't need to specifically ignore those files.

Reply:
ok! thanks